mitodl/mitxonline
# Order history and order receipt accessible when logged out

#### Pre-Flight checklist

#### What are the relevant tickets?
#673 
#### What's this PR do?
Allows only users who are logged in the ability to view the `orders/history/` and `orders/receipt/` pages.  If those pages are accessed by a user who is not logged in, then the user will be redirected to the sign in page.

#### How should this be manually tested?
Navigate to /orders/history/ as a user that isn't logged in. You should be redirected to the sign in page.

Navigate to the order history page after logging in. You should see the order history as per normal.
